mypy/messages.py

```diff
@@ -1601,9 +1601,10 @@ def overload_inconsistently_applies_decorator(self, decorator: str, context: Con
 
     def overloaded_signatures_overlap(self, index1: int, index2: int, context: Context) -> None:
         self.fail(
-            "Overloaded function signatures {} and {} overlap with "
-            "incompatible return types".format(index1, index2),
+            f"Overloaded function signatures {index1} and {index2} overlap with incompatible return types",
+            # "See https://mypy.readthedocs.io/en/stable/more_types.html#type-checking-the-variants.",
```

Comment:
Let's just remove this, if we don't end up adding the note this will be confusing for future readers of the code.